tests/Jellyfin.Networking.Tests/IPNetAddressTests.cs

```diff
@@ -1,49 +0,0 @@
-using FsCheck;
-using FsCheck.Xunit;
-using MediaBrowser.Common.Net;
-using Xunit;
-
-namespace Jellyfin.Networking.Tests
-{
-    public static class IPNetAddressTests
-    {
-        /// <summary>
-        /// Checks IP address formats.
-        /// </summary>
-        /// <param name="address">IP Address.</param>
-        [Theory]
-        [InlineData("127.0.0.1")]
-        [InlineData("fd23:184f:2029:0:3139:7386:67d7:d517")]
-        [InlineData("fd23:184f:2029:0:3139:7386:67d7:d517/56")]
-        [InlineData("[fd23:184f:2029:0:3139:7386:67d7:d517]")]
-        [InlineData("fe80::7add:12ff:febb:c67b%16")]
-        [InlineData("[fe80::7add:12ff:febb:c67b%16]:123")]
-        [InlineData("fe80::7add:12ff:febb:c67b%16:123")]
-        [InlineData("[fe80::7add:12ff:febb:c67b%16]")]
-        [InlineData("192.168.1.2/255.255.255.0")]
-        [InlineData("192.168.1.2/24")]
-        public static void TryParse_ValidIPStrings_True(string address)
```

Comment:
Are these tests no longer necessary or do they simply not fit into the new code?

Reply:
Those were tests for a data class which we don't use anymore.

Reply:
I just want to make sure that the tests are 1:1 so we don't introduce very subtle bugs that no one knows how to fix

Reply:
I will readd them and just test them against the .NET classes.

I'm quite sure some of the ones mentioned here are not handled properly. But from experience I can also say that especially the IPv6 stuff did not work as intended with the current implementation either.

Reply:
I moved over to my Parser function, which handles IP notations with subnets as subnets and simple IPs as subnets with a single IP scope.